main.py

```diff
@@ -6,26 +6,26 @@
 import tweepy
 
 # Loading twitter credentials
-consumer_key, consumer_secret, access_token, access_token_secret = load_twitter_env()
-
-# Authenticate to Twitter using Tweepy
-auth = tweepy.OAuthHandler(consumer_key, consumer_secret)
-auth.set_access_token(access_token, access_token_secret)
-
-# Connect to the TWITTER API
-api = tweepy.API(auth, wait_on_rate_limit=True, wait_on_rate_limit_notify=True)
+consumer_key, consumer_secret, access_token, access_token_secret, bearer_token = load_twitter_env()
 
+# Connect to the TWITTER Client
+client = tweepy.Client(bearer_token=bearer_token,
+                       consumer_key=consumer_key, 
+                       consumer_secret=consumer_secret, 
+                       access_token=access_token, 
+                       access_token_secret=access_token_secret)
 def reminder():
    while True:
      with open('reminders.txt') as f: lines = f.readlines()
-     
+
      # Select a random line from the reminders file.
-     lines = [line[1:] for line in lines if line[0] == "-"]
+     lines = [line[3:] for line in lines if len(line) > 2 and line[2] == "-"]
```

Comment:
I feel like there might be a better solution here.

Maybe check if the first non-whitespace character is the hyphen or not, and if it is, remove all the whitespaces around it?

Reply:
So strip the line of white space and then check if the first character is a hyphen? That would work. But I'm not sure what you mean by "remove all the whitespaces around it". Still, I'll try making an edit.

Alternately, would it be better to use Python's yaml parser?

Reply:
Yes it might be better to use Python's YAML parser!